header.txt

```diff
@@ -1,5 +1,5 @@
 Artifactural
-Copyright (c) 2018-2021.
```

Comment:
Wow this hasn't been built in quite some time. 
We've moved to the SPDX header format in modern projects for this exact reason.
Having times in the header isn't necessary anymore.
https://github.com/MinecraftForge/InstallerTools/blob/master/LICENSE-header.txt